package.json

```diff
@@ -40,7 +40,7 @@
   "scripts": {
     "clean": "rimraf lib",
     "format": "find src -name '*.ts' -not -path 'src/test/fixtures/*' -not -path 'src/test/goldens/*' | xargs clang-format --style=file -i",
-    "build": "npm run clean && tsc",
+    "build": "rm -f node_modules/parse5/lib/index.d.ts && npm run clean && tsc",
```

Comment:
You should also be able to the the `"paths"` compiler option, like:

```json
    "paths": {
      "parse5": [
        "node_modules/@types/parse5",
        "*"
      ]
    }
```

Reply:
Oh nice, that worked, thanks! https://github.com/Polymer/gen-typescript-declarations/pull/114

I guess we should try to get to parse5@5 which doesn't ship its own typings at all.